tests/erc-7589/revoke-handler.test.ts

```diff
@@ -89,23 +89,23 @@ describe('ERC-7589 RoleRevoked Handler', () => {
       findOrCreateRolesRegistry(rolesRegistry),
       granteeUser,
       RoleAssignmentId,
-      commitmentId.toString(),
+      depositId.toString(),
     )
     assert.fieldEquals('RoleAssignment', _id, 'expirationDate', '0')
   })
 
   test('should revoke multiple roles for the same NFT', () => {
-    const commitmentId1 = BigInt.fromI32(2)
-    const commitmentId2 = BigInt.fromI32(3)
-    const commitmentId3 = BigInt.fromI32(4)
+    const tokenCommitmentId1 = BigInt.fromI32(2)
```

Comment:
those are `depositIds`

```suggestion
    const depositId1 = BigInt.fromI32(2)
```